addons/mod_tool/dock.tscn

```diff
@@ -503,14 +505,69 @@ script = ExtResource( 2 )
 [node name="ValidationDelay" type="Timer" parent="TabContainer/Default Configuration/VBox/ConfigEditor"]
 one_shot = true
 
+[node name="Popup" type="WindowDialog" parent="."]
+unique_name_in_owner = true
+visible = true
+anchor_left = 0.5
+anchor_top = 0.5
+anchor_right = 0.5
+anchor_bottom = 0.5
+margin_left = -200.0
+margin_top = -75.0
+margin_right = 200.0
+margin_bottom = 75.0
+window_title = "Create new mod"
+
+[node name="CreateMod" parent="Popup" instance=ExtResource( 6 )]
+unique_name_in_owner = true
+
+[node name="Settings" parent="Popup/CreateMod" index="0"]
+margin_right = 390.0
+margin_bottom = 140.0
+
+[node name="Scroll" parent="Popup/CreateMod/Settings" index="0"]
+margin_right = 380.0
+margin_bottom = 77.0
+
+[node name="VBox" parent="Popup/CreateMod/Settings/Scroll" index="0"]
+margin_right = 380.0
+
+[node name="ModId" parent="Popup/CreateMod/Settings/Scroll/VBox" index="0"]
+margin_right = 380.0
+
+[node name="HBox" parent="Popup/CreateMod/Settings/Scroll/VBox" index="1"]
+margin_right = 380.0
+
+[node name="Label" parent="Popup/CreateMod/Settings/Scroll/VBox/HBox" index="0"]
+margin_right = 188.0
+
+[node name="ThunderstoreExport" parent="Popup/CreateMod/Settings/Scroll/VBox/HBox" index="1"]
+margin_left = 192.0
+margin_right = 380.0
```

Comment:
I don't think we need that in the create window

Reply:
Hmm I have this in my notes:

If Thunderstore Export == true

- [ ]  Add *[README.md](http://readme.md/)* to root dir
- [ ]  Add *manifest.json* to root dir
- [ ]  Add *CHANGELOG.md* to root dir
- [ ]  Add *icon.png* to root dir

Reply:
We can hide it for now.
*[9c1892b](https://github.com/GodotModding/godot-mod-tool/pull/24/commits/9c1892bc47a7fc01846a6ebb54104e6a74c51adf)*

Reply:
the checkbox in the bottom panel should suffice. It should be just an extra processing step that happens on export, not for the mod structure. We can just add those files to the mod directory, like our manifest and move all the files needed by thunderstore one level out before zipping. That's why I placed the checkbox in the export dialog, not with mod creation

Reply:
We decided to always add the thunderstore files to mod directory
*[228404f](https://github.com/GodotModding/godot-mod-tool/pull/24/commits/228404f3ac2630b7d3c1f9fd97e01fc77f3aec6e)*